IBX-5450 Fixes to Commerce procedures in User Doc (#200)
ezsystems/user-documentation: docs/commerce/payment/create_payment_methods.md

If your [user role](work_with_permissions.md) includes the `Payment method/Create` permission, you can create payment methods. 
With the `Payment method/Edit` permission, you can modify existing ones.

> Payment method definitions describe the way customers of an e-commerce presence pay for their orders during the checkout process.

Reviewer: ```suggestion
Payment methods describe the way shop customers pay for their orders during the checkout process.
```
Is there any business reason to not plainly say "shop"? "e-commerce presence" is too fancy by far, and really meaningful

![Creating a new payment method](create_new_payment_method.png)

4\. Toggle the **Availability** switch on, so that store customers can select this shipping method during checkout.

5\. Click **Create** to save your changes.

## Edit an existing payment method

1\. In the left panel, go to **Commerce** -> **Payment methods**.

![Payment methods list](payment_methods_list.png)

2\. Find the payment method that you intend to edit by using the search field and filters.

> 3\. If you are deleting a shipping method, select a box next to its name and click **Delete**, which concludes the procedure. Otherwise, skip to step 4.

Reviewer: I'd move this to a separate section, otherwise a note. The procedure is "edit payment method", so mentioning deleting is interrupting the flow.

!!! note "Payment methods for existing orders"

>     You cannot delete a payment method if it is active or unpaid orders exist that use this specific payment method. You must first deactivate the method that you want to delete by toggling the **Availability** switch off.

Reviewer: ```suggestion
    You cannot delete a payment method if it is active or if it is used by unpaid orders.
    You must first deactivate the method that you want to delete by toggling the **Availability** switch off.
```
